MapboxNavigation/NavigationView.swift

```diff
@@ -227,6 +228,35 @@ open class NavigationView: UIView {
         instructionsBannerView.delegate = delegate
         statusView.delegate = delegate
     }
+    
+    /**
+     Updates the instructions banner for a given `RouteProgress`.
+     */
+    public func update(for routeProgress: RouteProgress) {
+        let stepProgress = routeProgress.currentLegProgress.currentStepProgress
+        let distanceRemaining = stepProgress.distanceRemaining
+        
+        guard let visualInstructions = stepProgress.remainingVisualInstructions else { return }
+        
+        for visualInstruction in visualInstructions {
+            if stepProgress.distanceRemaining <= visualInstruction.distanceAlongStep || stepProgress.visualInstructionIndex == 0 {
+                
+                nextBannerView.hide()
+                lanesView.update(for: visualInstruction, currentStepProgress: routeProgress.currentLegProgress.currentStepProgress)
+                
+                if lanesView.isHidden {
```

Comment:
Logic like this should not occur. We should be render whatever we have to this view since we know we will never have both a lane view and a then view.

Reply:
@bsudekum I concur. Was just being extra-cautious.